Components/ReusableComponent/functions.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Flynt\Components\ReusableComponent;
+
+use Timber\Timber;
+
+add_filter('Flynt/addComponentData?name=ReusableComponent', function ($data) {
+    if (!empty($data['reusableId'])) {
+        $data['post'] = Timber::get_post($data['reusableId']);
+    }
+
+    return $data;
+});
+
+function getACFLayout()
+{
+    return [
+        'name' => 'ReusableComponent',
+        'label' => '<i class="dashicons dashicons-controls-repeat"></i> Reusable',
```

Comment:
The icon / text position looks too much to the top in the component dropdown. Any chance to fix that? Also would consider putting the icon at the end, and / or moving the component position to the very end of the list.

![Bildschirmfoto 2020-09-29 um 20 12 07](https://user-images.githubusercontent.com/102498/94599036-24bf6000-0290-11eb-933e-aacf600fbe73.png)